tests/yam/agama/agama_auto.pm

```diff
@@ -15,6 +15,12 @@ sub run {
     my $reboot_page = $testapi::distri->get_reboot_page();
 
     $reboot_page->expect_is_shown(timeout => 1200);
+
+    select_console 'root-console';
+    Yam::Agama::agama_base::upload_agama_logs();
+    Yam::Agama::agama_base::upload_system_logs();
+
+    select_console 'installation';
```

Comment:
this is not needed anymore, it is doing it behind the scene.

Reply:
Removed, thanks.